docs/acme-divergences.md

```diff
@@ -9,8 +9,7 @@ Presently, Boulder diverges from the RFC 8555 ACME spec in the following ways:
 
 Boulder supports POST-as-GET but does not mandate it by default for requests
 that simply fetch a resource (certificate, order, authorization, or challenge).
-This behavior is configurable with a flag: Let's Encrypt's Staging environment
-does mandate POST-as-GET, while the Production environment does not.
+Neither the Let's Encrypt's Staging nor Production environment mandate POST-as-GET.
```

Comment:
nit: maybe just remove this sentence entirely, and modify the preceding sentence to remove "by default". Then it's just straightforwardly stated that Boulder does not mandate post-as-get.